etl/steps/data/garden/AI/2023-06-08/yougov_job_automation.py

```diff
@@ -0,0 +1,51 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("yougov_job_automation.start")
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("yougov_job_automation"))
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["yougov_job_automation"]
+    df = pd.DataFrame(tb)
+
+    # Create a date column (counting days since 2021-01-01)
+    df["days_since_2021"] = (
+        pd.to_datetime(df["year"].astype(str), format="%Y-%m-%d") - pd.to_datetime("2021-01-01")
+    ).dt.days
+    df = df.drop("year", axis=1)
+    df.rename(columns={"days_since_2021": "year"}, inplace=True)
```

Comment:
If the column represents days, then it should not be called "year". If you are doing this because you need a "year" column in grapher, you should do this renaming in the grapher step.